test/test_common/test_base.h

```diff
@@ -28,7 +30,9 @@ class TestBase : public ::testing::Test {
 // Templatized version of TestBase. See above notes.
 template <class T> class TestBaseWithParam : public ::testing::TestWithParam<T> {
 public:
-  ~TestBaseWithParam() { TestBase::checkSingletonQuiescensce(); }
+  ~TestBaseWithParam() {
+    RELEASE_ASSERT(TestBase::checkSingletonQuiescensce(), "active singletons exist");
```

Comment:
OK this is good...I was thinking about ASSERT_TRUE(...) or EXPECT_TRUE(...) here so that further tests would continue to run. But maybe it's hard to reason about more tests when there are leaks in the current one.